packages/instant/test/util/asset_data.test.ts

```diff
@@ -1,17 +0,0 @@
-import { assetDataUtil } from '../../src/util/asset_data';
```

Comment:
Think it'd be worth it to convert these tests to the new signature instead of removing them?